gce/burst-training/README.md

```diff
@@ -0,0 +1,242 @@
+# Compute Engine burst training
+
+[Google Compute Engine](https://cloud.google.com/compute/) gives you access to
+very powerful virtual machines which you can use for as much or, more
+importantly for most data science use cases, as little time as you need.
+
+Common tasks that might take tens of minutes or even hours on your personal can
+be completed in seconds on a machine with 64 cores and almost half a terabyte of
+memory, and this can be done at a cost of under $1.
+
+This makes Google Cloud Platform (or any equivalent cloud infrastructure for
+that matter) a powerful tool for model training. The idea is simple:
+
+1. Write a trainer locally, testing it on a small sample of your training data
+   to make sure that the training job is sound in principle (that it has no
+   syntax errors and such).
+
+2. Package the trainer and deploy it to a virtual machine (VM) instance with
+   many, many CPU cores and as much RAM as necessary.
```

Comment:
As you said, I think this is low priority - we should probably only work on this if we keep hearing questions about it from our users.

Reply:
Incidentally, I think that the cross-validation multiprocessing is broken as I ran into some issues with it while working on this sample. Will have to investigate a little more carefully later.

Reply:
What were the issues you ran into?

Reply:
The training job hangs if I specify `n_jobs` > 1 in the `RandomizedSearchCV`. I tried this with `sklearn.ensemble.GradientBoostingClassifier` and did not have this issue. But it's not clear whether that is because `GradientBoostingClassifier` does not parallelize or because `xgboost` did something wrong.

Reply:
Given this, I am okay switching from `xgboost` to scikit-learn's `GradientBoostingClassifier`.

Reply:
Just say the word @dizcology (or anyone)

Reply:
I haven't had the chance to try this with `GradientBoostingClassifier`.  Does it not hang and successfully complete the training?  If so let's use it for now until we figure out what's wrong with `xgboost`.

Does `xgboost` always hang or sometimes?

Reply:
`xgboost` always hangs, `GradientBoostingClassifier` does not hang.

Reply:
Fixed. xgboost classifier replaced with GradientBoostingClassifier.